back/src/api/type/infraction-severity.type.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, Int, ObjectType } from "type-graphql"
+import { SeverityModel } from "../../domain/model"
+
+@ObjectType()
+export class Severity implements SeverityModel {
+  
+  @Field(() => Int, {
+    description: "Numbered grade took accordingly to infraction severity",
+    defaultValue: 3 | 5 | 7 | 12,
```

Comment:
aceitar apenas um valor exemmplo:

```suggestion
    defaultValue: 3,
```